MovingBlocks/TerasologyLauncher
# Indicate installed packages

### Contains
Adds a checkmark (:heavy_check_mark:) beside all installed packages, and hides it again if the package is removed. Works with the help of custom `VersionListCell` class to render the icon. It also improves `PackageDatabase` for compatibility purposes.
![2019-12-03-225641_969x563_scrot](https://user-images.githubusercontent.com/14859252/70074284-5f308000-1620-11ea-9156-867b5b7d18c2.png)
Fixes #474.

### How to test
1. Start the launcher normally.
2. Select any package and install/remove it.
3. The green checkmarks should be shown accordingly.
